### PR TITLE
feat: Propagate git commit ID for sequence in CloudEvent context

### PR DESCRIPTION
**Pull request by odubajDT**
Signed-off-by: odubajDT <ondrej.dubaj@dynatrace.com>

Fixes https://github.com/keptn/keptn/issues/6347

Binded with https://github.com/keptn/go-utils/pull/374

### CODE REVIEW

File: cloudevents.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ In Keptn, events have a payload structure as follows (*Note:* The `triggeredid`
     "specversion",
     "time",
     "triggeredid",
+    "gitcommitid",
```
Comment (johannes-b) [inline cloudevents.md:30]: I would like to ask why the `gitcommitid` is part of the event meta-data and not an additional property in `data` like `project`, `stage`, `services`.

Comment (odubajDT) [reply cloudevents.md:30]: I understand that `gitcommitid` is a pointer to specific version of the resources (for example shipyard file), which should be used unchanged during the whole sequence. I actually think this information is more like a meta-data, as it defines a scope of a certain git-commit-id. I do not consider it as a data like information about project, stage, etc. Also in the description of the issue, there was a task to provide this information next to the context attributes like `triggeredid`. Please see the issue here: keptn/keptn#6347

Comment (johannes-b) [reply cloudevents.md:30]: @odubajDT thanks for the clarification and the reasoning behind that. 

I just want to let you know that it is most likely the case that the git commit will change during the sequence execution when we switch the stage. The reason is that we store stage config on different branches and one branch can be ahead of another branch resulting in a different commit id.

Comment (odubajDT) [reply cloudevents.md:30]: Ok, so do you suggest to change this ? The actual code was already merged, only the documentation is left to merge.

Comment (johannes-b) [reply cloudevents.md:30]: No, no need to change. Looks good!

Comment (odubajDT) [reply cloudevents.md:30]: thank you :) can you approve it then please ?

**Review by johannes-b — APPROVED** [review]
lgtm

**Pull request merged**